multiplayer/src/components/GamePage.tsx

```diff
@@ -52,49 +58,50 @@ export default function Render(props: GamePageProps) {
             {state.gameState?.gameMode && (
                 <div className="tw-flex tw-flex-col tw-items-center">
                     {state.playerSlot && <ArcadeSimulator />}
-                    <div className="tw-flex tw-flex-row tw-space-x-2 tw-w-full">
-                        {state.gameState?.joinCode && (
-                            <div>
-                                {state.gameState?.joinCode &&
-                                    `${lf("Join Code")}: ${
-                                        state.gameState?.joinCode
-                                    }`}
-                                <button
-                                    onClick={copyJoinCode}
-                                    title={lf("Copy Join Code")}
-                                >
-                                    <div className="tw-text-sm tw-ml-1">
-                                        {!copySuccessful && (
-                                            <FontAwesomeIcon
-                                                icon={faCopy}
-                                                className="hover:tw-scale-105 tw-mb-[0.1rem]"
-                                            />
-                                        )}
-                                        {copySuccessful && (
-                                            <FontAwesomeIcon
-                                                icon={faCheck}
-                                                className="tw-text-green-600 tw-mb-[0.1rem]"
-                                            />
-                                        )}
-                                    </div>
-                                </button>
-                            </div>
-                        )}
-                        <div className="tw-flex-grow" />
+                    <div className="tw-flex tw-flex-row tw-w-full tw-px-2 tw-items-center tw-justify-between">
+                        <button
```

Comment:
Let's use react-common's Button here. It's set up with all the needed accessibility props. You might want to inject the font awesome icon as a JSX.Element to the `label` prop.